x-pack/plugin/esql/compute/src/main/generated/org/elasticsearch/compute/aggregation/CountDistinctBooleanAggregatorFunction.java

```diff
@@ -28,14 +29,18 @@ public final class CountDistinctBooleanAggregatorFunction implements AggregatorF
 
   private final List<Integer> channels;
 
+  private final DriverContext driverContext;
+
   public CountDistinctBooleanAggregatorFunction(List<Integer> channels,
-      CountDistinctBooleanAggregator.SingleState state) {
+      CountDistinctBooleanAggregator.SingleState state, DriverContext driverContext) {
     this.channels = channels;
     this.state = state;
+    this.driverContext = driverContext;
   }
 
-  public static CountDistinctBooleanAggregatorFunction create(List<Integer> channels) {
-    return new CountDistinctBooleanAggregatorFunction(channels, CountDistinctBooleanAggregator.initSingle());
+  public static CountDistinctBooleanAggregatorFunction create(List<Integer> channels,
+      DriverContext driverContext) {
+    return new CountDistinctBooleanAggregatorFunction(channels, CountDistinctBooleanAggregator.initSingle(), driverContext);
```

Comment:
Could you flip the order of these? It's not a big deal, but I feel like the context is "bigger" than the channels and the state.